league_rpc_linux/lcu/LCU.py

```diff
@@ -0,0 +1,76 @@
+import requests
+from league_rpc_linux.processes.process import get_process_command
+
+
+class LCU:
+    """
+    League Client Update (LCU) API class.
+    """
+
+    def __init__(self):
+        self.lcu_connect = None
+        self.lcu_connect_port = None
+        self.lcu_connect_password = None
+        self.lcu_connect_protocol = "https"
+        self.lcu_connect_host = "127.0.0.1"
+        self.get_lcu_connect_details(proc_name="CrBrowserMain")
```

Comment:
This is too generic, and won't work on windows. Opt into using lcu-driver instead.

My PR on that project makes it possible to find LeagueClientUx.exe, even if the process name is CrBrowserMain

Reply:
invalid - updated to lcu-driver